python/ql/src/experimental/semmle/python/frameworks/Stdlib.qll

```diff
@@ -245,29 +255,28 @@ private module Stdlib {
    * A call to `subprocess.Popen` or helper functions (call, check_call, check_output, run)
    * See https://docs.python.org/3.8/library/subprocess.html#subprocess.Popen
    */
-  private class SubprocessPopenCall extends SystemCommandExecution::Range {
-    CallNode call;
+  private class SubprocessPopenCall extends SystemCommandExecution::Range, DataFlow::CfgNode {
+    override CallNode node;
 
     SubprocessPopenCall() {
-      call = this.asCfgNode() and
       exists(string name |
         name in ["Popen", "call", "check_call", "check_output", "run"] and
-        call.getFunction() = subprocess_attr(name).asCfgNode()
+        node.getFunction() = subprocess_attr(name).asCfgNode()
       )
     }
 
     /** Gets the ControlFlowNode for the `args` argument, if any. */
     private ControlFlowNode get_args_arg() {
-      result = call.getArg(0)
+      result = node.getArg(0)
       or
-      result = call.getArgByName("args")
+      result = node.getArgByName("args")
```

Comment:
Could be `result in [node.getArg(0), node.getArgByName("args")]`
(It will not let me make a suggestion on a "deleted" line..)

Reply:
odd, I'll go through and make the changes manually then. Thanks for pointing them out :+1: